python/test/polars/test_truncation.py

```diff
@@ -0,0 +1,213 @@
+import opendp.prelude as dp
+import polars as pl
+import pytest
+import re
+
+
+def test_lazyframe_bounded_dp_truncation():
```

Comment:
These tests (except for the last) are moved from the (too big) Polars test file.